inception-app-webapp/src/main/java/de/tudarmstadt/ukp/inception/metrics/InceptionMetrics.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020
+ * Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.metrics;
+
+public interface InceptionMetrics
+{
+    /**
+     * Retrieve the total number of currently active (i.e. logged in) users
+     */
+    public long getActiveUsersTotal();
```

Comment:
You could call these methods `countTotalActiveUsers`, same with the other getter names

Reply:
If you search the code base (WebAnno+INCEpTION), you'll find many methods called `blahCount()` or `getBlahCount()`.

Reply:
I chose these names because the metrics will then automatically be exported as `ActiveUsersTotal` , the total suffix is a naming convention for prometheus metrics and my exporter config will change it to snake-case. I'm sure that there must be a way to customize metrics names in JMX independent of method name, I could look into it if this is really important.